bagitobjecttransfer/recordtransfer/admin.py

```diff
@@ -90,15 +92,68 @@ class CustomUserAdmin(UserAdmin):
         ),
     )
 
+    @sensitive_post_parameters_m
+    def user_change_password(self, request, id, form_url=''):
+        """ Send a notification email when a user's password is changed. """
+        response = super().user_change_password(request, id, form_url)
+        user = self.get_object(request, unquote(id))
+        form = self.change_password_form(user, request.POST)
+        if form.is_valid() and request.method == 'POST':
+            context = {
+                'subject': "Password updated",
+                'changed_item': "password",
+                'changed_status': "updated"
+            }
+            send_user_account_updated.delay(user, context)
+        return response
+
     def save_model(self, request, obj, form, change):
-        # TODO: We may want to notify the modified user (by sending an email) if any major changes
-        #       are made to their account.
+        """ Enforce superuser permissions checks and send notification emails for other account updates. """
         if change and obj.is_superuser and not request.user.is_superuser:
             messages.set_level(request, messages.ERROR)
             msg = 'Non-superusers cannot modify superuser accounts.'
             self.message_user(request, msg, messages.ERROR)
         else:
             super().save_model(request, obj, form, change)
+            if change and (not obj.is_active or self._alert_user_changed(form)):
+                if not obj.is_active:
+                    context = {
+                        'subject': "Account Deactivated",
+                        'changed_item': "account",
+                        'changed_status': "deactivated"
+                    }
+                else:
+                    context = {
+                        'subject': "Account updated",
+                        'changed_item': "account",
+                        'changed_status': "updated",
+                        'changed_list': self._get_changed_message(form.changed_data, obj)
+                    }
+
+                send_user_account_updated.delay(obj, context)
+
+    @staticmethod
+    def _get_changed_message(changed_data: list, user: User):
+        """ Generate a list of changed status message for certain account details. """
+        message_list = list()
+        for item in changed_data:
+            message = None
+            change = None
+            if item == "is_superuser":
+                message = "Superuser privileges"
+                change = user.is_superuser
+            elif item == "is_staff":
+                message = "Staff privileges"
+                change = user.is_staff
+            if message is not None:
+                message += " have been {} your account".format("added to" if change else "removed from")
+                message_list.append(message)
+        return message_list
```

Comment:
These also need gettext(), but the way this is written is not conducive to translation. For example, what if in some language it makes grammatic sense to say "Added to your account is Staff privileges"? There are only four cases here, it should be OK to hard code them:

```python

if item == 'is_superuser' and user.is_superuser:
    msg = gettext('Superuser privileges have been added to your account')
    message_list.append(msg)
elif item == 'is_superuser' and not user.is_superuser:
    msg = gettext('Superuser privileges have been removed from your account')
    message_list.append(msg)
```

And so on